addons/storyshots/storyshots-puppeteer/.eslintrc.js

```diff
@@ -0,0 +1,10 @@
+const error = 2;
+const warn = 1;
+const ignore = 0;
+
+module.exports = {
+  rules: {
+    'import/no-unresolved': ignore,
```

Comment:
```
'settings': {
  'import/core-modules': ['puppeteer']
}
```

https://github.com/benmosher/eslint-plugin-import#importcore-modules

Reply:
thanks, that works!